docs/dev/analysis/tbl-cell-borders.rst

```diff
@@ -0,0 +1,92 @@
+
+Table Cell Borders
+------------------
+
+In PowerPoint, a table cell can have borders on all 4 sides, plus two diagonal lines (bottom left to top right and
+top left to bottom right) for a total of 6 possible borders, each specified by a child element of the table cell
+properties element.
+
+Each border is further customized by a set of attributes and child elements common to all borders (`LineFormat`).
+
+In order to provide `LineFormat` functionality, each border `_ln` object is wrapped in a `_TableBorderLine`,
+which implements the `get_or_add_ln` method, and can be then wrapped by `LineFormat`.
+
+Protocol
+--------
+
+**Starting with a default cell** A new cell has no borders or style.
+New borders can be created by accessing the borders property::
+
+    >>> table = shapes.add_table(rows, columns, left, top, width, height).table
+    >>> cell = table.rows[0].cells[0]
+    >>> cell
+    <pptx.shapes.table._Cell object at 0x...>
+    >>> cell.borders
+    <pptx.shapes.table._TableBorders object at 0x...>
+
+**Create a border** A cell border is created by applying line width and color for its `LineFormat`.
+This can be done with any one of the 6 cell borders::
+
+    >>> cell.borders.top.width = Pt(1)
+    >>> cell.borders.top.color.rgb = RGBColor(0, 0, 0)
+    >>> cell.borders.top._ln.xml
+    <a:lnT w="12700">
+      <a:solidFill>
+        <a:srgbClr val="000000"/>
+      </a:solidFill>
+    </a:lnT>
+    >>> cell._tc.xml
+    <a:tc>
+      <a:txBody>
+        <a:bodyPr/>
+        <a:lstStyle/>
+        <a:p/>
+      </a:txBody>
+      <a:tcPr>
+        <a:lnT w="12700">
+          <a:solidFill>
+            <a:srgbClr val="000000"/>
+              </a:solidFill>
+        </a:lnT>
+      </a:tcPr>
+    </a:tc>
+
+**Apply different line styles** Cell border work with `LineFormat`, so any other line style can also be applied::
+
+    >>> cell.borders.top.dash_style = MSO_LINE_DASH_STYLE.DASH_DOT_DOT
+    >>> cell.borders.top._ln.xml
+    <a:lnT w="12700">
+      <a:solidFill>
+        <a:srgbClr val="000000"/>
+      </a:solidFill>
+      <a:prstDash val="lgDashDotDot"/>
+    </a:lnT>
+
+
+Related Schema Definitions
+--------------------------
+
+::
+
```

Comment:
Insert the complex type definition for table cell here, so we can remember where we get the cell properties element from and what its tag name and cardinality is. I expect you can just copy/paste that in from the table cell analysis page or thereabouts.